includes/class-wc-calypso-bridge-setup.php

```diff
@@ -166,6 +166,25 @@ public function is_theme_installed( $theme ) {
 	public function register_payments_welcome_page() {
 		global $menu;
 
+		// WooCommerce must be active.
+		if ( !is_plugin_active( 'woocommerce/woocommerce.php' ) ) {
+			return;
+		}
+
+		// WC Payment must not be installed.
+		if ( array_key_exists( 'woocommerce-payments/woocommerce-payments.php', get_plugins() ) ) {
+			return;
+		}
+
+		// User country must be US.
```

Comment:
Nitpick: It's more correct to refer to the store”s country. 
```suggestion
		// Store country must be US.
```

Reply:
Thank you!

Reply:
Updated in https://github.com/Automattic/wc-calypso-bridge/pull/703/commits/4059fe445ca392e315163040c95a8ee6e19c1f98